src/modules/ApplicationsPage/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Typography } from 'm3ntorship-ui';
+import Mentee from 'public/images/applications-page/mentee.svg';
+import Mentor from 'public/images/applications-page/mentor.svg';
+import { ReactElement, useState } from 'react';
+
+import ApplicationsTab from '@/modules/ApplicationsPage/components/ApplicaitonsTab';
+import ApplicationsHeader from '@/modules/ApplicationsPage/components/ApplicationsHeader';
+import MenteeApplication from '@/modules/ApplicationsPage/components/MenteeApplication';
+import MentorApplication from '@/modules/ApplicationsPage/components/MentorApplication';
+
+export default function ApplicationsPage(): ReactElement {
+  const [activeTab, setActiveTab] = useState<string>('mentee');
```

Comment:
the first view in this page on [Figma](https://www.figma.com/file/5KpDnPPmrZhudGESduPhq3/M3ntorship-Website?node-id=1089%3A8106) only contains the tabs, so this should be null in the initial render
```suggestion
  const [activeTab, setActiveTab] = useState<null | 'mentee' | 'mentor'>(null);
```
and would be better to create an enum for them